root/communication-rmi/src/main/java/at/fhv/itb5c/rmi/client/RMIClient.java

```diff
@@ -27,7 +27,11 @@ public void startUp() {
 	}
 
 	public void close() {
-		throw new UnsupportedOperationException();
+		// Note: This will fail, if something on the outside still holds a reference to _applicationFacadeRMIStub.
```

Comment:
And then we'd have to remember the proxy and set its attribute *_applicationFacadeRMIStub* to null. Yes, that's a nice way to do that.
However, we soon have to refactor that stuff anyway when working with ejb. (--> additional abstraction). So let's leave it for now how it is.